instrumentation/elasticsearch/elasticsearch-api-client-8.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/elasticsearch/apiclient/v8_0/ElasticsearchEndpointMap.java

```diff
@@ -0,0 +1,774 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.elasticsearch.apiclient.v8_0;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ElasticsearchEndpointMap {
+  private static final Set<String> searchEndpoints =
+      Stream.of(
```

Comment:
I think we usually use `new HashSet<>(Arrays.asList(...))`, but this should also work